src/composables/useBorrowerProfileData.js

```diff
@@ -0,0 +1,174 @@
+	const originalLanguage = computed(() => loan.value?.originalLanguage ?? null);
+	const descriptionInOriginalLanguage = computed(() => loan.value?.descriptionInOriginalLanguage ?? '');
+	const reviewer = computed(() => (loan.value && 'reviewer' in loan.value ? loan.value.reviewer : null));
+	const partnerName = computed(() => (loan.value && 'partnerName' in loan.value ? loan.value.partnerName : ''));
+	const video = computed(() => loan.value?.video ?? null);
+	const businessName = computed(() => (loan.value && 'businessName' in loan.value ? loan.value.businessName : ''));
+	const businessDescription = computed(() => (
+		loan.value && 'businessDescription' in loan.value ? loan.value.businessDescription : ''
+	));
+	const dualStatementNote = computed(() => (
+		loan.value && 'dualStatementNote' in loan.value ? loan.value.dualStatementNote : ''
+	));
+	const moreInfoAboutLoan = computed(() => (
+		loan.value && 'moreInfoAboutLoan' in loan.value ? loan.value.moreInfoAboutLoan : ''
+	));
+	const purpose = computed(() => (loan.value && 'purpose' in loan.value ? loan.value.purpose : ''));
+	const sector = computed(() => loan.value?.sector ?? null);
+	const yearsInBusiness = computed(() => (
+		loan.value && 'yearsInBusiness' in loan.value ? loan.value.yearsInBusiness : ''
+	));
+	const socialLinks = computed(() => (loan.value && 'socialLinks' in loan.value ? loan.value.socialLinks : null));
+
+	const unreservedAmount = computed(() => loan.value?.unreservedAmount);
+	const fundraisingPercent = computed(() => loan.value?.fundraisingPercent ?? 0);
+	const timeLeft = computed(() => loan.value?.fundraisingTimeLeft ?? '');
+
+	const pfpMinLenders = computed(() => loan.value?.pfpMinLenders ?? 0);
+	const lenders = computed(() => loan.value?.lenders ?? null);
+	const inPfp = computed(() => loan.value?.inPfp ?? false);
+	const loanLenderRepaymentTerm = computed(() => loan.value?.lenderRepaymentTerm ?? 0);
+	const loanTermLenderRepaymentTerm = computed(() => loan.value?.terms?.lenderRepaymentTerm ?? 0);
+	const repaymentInterval = computed(() => loan.value?.repaymentInterval ?? '');
+	const disbursalDate = computed(() => loan.value?.disbursalDate ?? '');
+	const terms = computed(() => loan.value?.terms ?? null);
+	const endorsement = computed(() => (loan.value && 'endorsement' in loan.value ? loan.value.endorsement : ''));
+	const loanAmount = computed(() => loan.value?.loanAmount || 0);
+	const paidAmount = computed(() => loan.value?.paidAmount || 0);
+	const comments = computed(() => loan.value?.comments?.values ?? []);
+	const teamCount = computed(() => loan.value?.teams?.totalCount);
+	const lenderCount = computed(() => loan.value?.lenders?.totalCount);
+
+	const loadBPData = loanDataId => {
+		if (!loanDataId || typeof loanDataId !== 'number') return;
+		// Store the current loan ID for filtering
+		currentLoanId.value = loanDataId;
+		try {
+			// Watch borrower profile side sheet query
+			watchLoanData({
```

Comment:
We also need to store the return of `watchLoanData` and unsubscribe in `clearBPData`